http/src/main.rs

```diff
@@ -134,8 +134,13 @@ fn main() {
     let mut rt = tokio::runtime::Runtime::new().expect("Failed to create event loop");
 
     rt.block_on(async move {
-        let opts: IpfsOptions<ipfs::TestTypes> =
-            IpfsOptions::new(home.clone().into(), keypair, Vec::new(), false);
+        let opts: IpfsOptions<ipfs::TestTypes> = IpfsOptions::new(
+            home.clone().into(),
+            keypair,
+            Vec::new(),
+            false,
+            Some("/ipfs/kad/1.0.0".into()),
```

Comment:
elsewhere with "/lan", but recommended not adding this at all: https://github.com/rs-ipfs/rust-ipfs/pull/256/files#r458760223

Reply:
I added `/lan` only in test scenarios; as mentioned before, seems I can leave it out for now